connections/sim/sim_connection.py

```diff
@@ -43,26 +46,27 @@ def __init__(self, firmwareDir, executableName):
 
         self._xbee = XBeeModuleSim()
         self._xbee.rocket_callback = self._send_radio_sim
-    
+
+        self._hw_sim = hw_sim
+
     def _rocket_handshake(self):
         assert self.stdout.read(3) == b"SYN"
         # Uncomment for FW debuggers, for a chance to attach debugger
         # input("Recieved rocket SYN; press enter to respond with ACK and continue\n")
         self.rocket.stdin.write(b"ACK")
         self.rocket.stdin.flush()
-        
+
     def send(self, data):
         self._xbee.send_to_rocket(data)
 
-    def _send_radio_sim(self, data):
-        packet = b"R"
-        packet += len(data).to_bytes(length=2, byteorder="big")
-        packet += data
-
+    def _send_sim_packet(self, id_, data):
+        packet = id_ + len(data).to_bytes(length=2, byteorder="big") + data
```

Comment:
Does the underscore in `id_` mean anything? Also if we cast the id to bytes in this function, we don't have to do `b"x"` everywhere.

Reply:
id is a built-in function, and Python style guide suggests appending an underscore to avoid naming conflicts with builtins.